manage-page-order.p6

```diff
@@ -0,0 +1,414 @@
+        }
+
+        # data collected, put in hash
+        # hash key: stripped-filename
+        #   subkey: page-order
+        #   subkey: title
+        %data{$fname}<title>      = $title;
+        %data{$fname}<page-order> = $order;
+    }
+}
+
+sub strip-comment($line is copy) {
+    say "DEBUG: in sub {&?ROUTINE.name}" if $debug;
+    my $idx = rindex $line, '#';
+    if $idx.defined {
+        $line = substr $line, 0, $idx;
+    }
+    return $line;
+}
+
+sub extract-page-order(:begin-pod-line($line)) {
+    say "DEBUG: in sub {&?ROUTINE.name}" if $debug;
+    my $order = '';
+    if $line ~~ /':page-order<' (.*) '>'/ {
+        $order = ~$0;
+    }
+    return $order;
+}
+
+sub extract-abbrev-block-line($line is copy, :$type) {
+    say "DEBUG: in sub {&?ROUTINE.name}" if $debug;
+    $line = strip-comment $line;
+    die "FATAL: Empty '$type' abbreviated block first line"
+        if $line !~~ /\S/;
+    my $value = '';
+    my $idx = rindex $line, $type;
+    if defined $idx {
+        my $len = $type.chars;
+        $value = substr $line, $idx+$len;
+    }
+    $value = $value.words.join(' ');
+
+    return $value;
+}
+
+
+sub help {
+    say "DEBUG: in sub {&?ROUTINE.name}" if $debug;
+    say qq:to/HERE/;
+    $usage
+
+    Modes:
+        update  - Uses the pod6 source files and the control file
+                  ($cfil) to create the .pod6 files with the data
+                  for the desired sort order.
+
+        control - Extracts data from original pod files and builds
+                  a control file with pertinent data for future operations.
+                  The control file is a master list of =TITLEs and base
+                  file names for sorting as desired.
+
+                  One of two options must be selected:
+
+                    refresh - updates title data for an existing control file
+                    new     - creates a new control file
+
+                  An existing control file ($cfil) cannot be
+                  overwritten without permission.
+
+    Options:
+        -r      - refresh (for mode 'control')
+        -n      - create  (for mode 'control')
+        -h      - extended help
+        -d      - debug
+
+    Note: The modes are selected by entering either the first letter
+          of the mode name or its complete name.
+    HERE
+    exit;
+}
+
+sub is-done(*@args) {
+    say "DEBUG: in sub {&?ROUTINE.name}" if $debug eq 'r';
+    # return False unless all @args elements are true
+    for @args -> $v {
+        return False if !$v;
+    }
+    return True;
+}
+
+sub write-group-page($group-char, $title, $page-order) {
+    my $fname = $todir ~ "/00-Group{$group-char}.pod6";
+    my $fh = open $fname, :w;
+
+    # x2588 big solid block <== using this char to make the separator
+    # x2501 solid thick horizontal line
+    $fh.print: qq:to/HERE/;
+        =begin pod :page-order<{$page-order}>
+        =TITLE $title
+        =SUBTITLE █████████████████████████████████████████
```

Comment:
Why are we doing unicode art when we can output html?

Reply:
Or what's the point of this?

Reply:
Until the html can be affected, the bar gives a good visual separator between groups. This step is intended to get the idea of specific grouping and sorting onto the screen with existing tools before the rendering tools are modified.

There is a thinner bar available, or other choices.

I also am investigating the travis pod failure...

Reply:
Fixed typo